### PR TITLE
Export rules disabled by multiple users

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Ability to export rules disabled by multiple users

Fixes #68

## Type of change

- New feature (non-breaking change which adds functionality)
- Unit tests (no changes in the code)
- Documentation update

## Testing steps

Done on CI

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: storage.go [hunk 1]
```diff
@@ -645,3 +653,37 @@ func writeColumnNames(writer *csv.Writer, colNames []string) error {
 	}
 	return nil
 }
+
+// ReadDisabledRules method reads rules disabled by more that one user
```
Comment (juandspy) [inline storage.go:657]: "that" -> "than"

File: storage_test.go [hunk 2]
```diff
@@ -216,3 +217,64 @@ func TestReadRecordCountOnError(t *testing.T) {
...
+	if err == nil {
+		t.Errorf("error was expected")
+	}
+
+	// connection to mocked DB needs to be closed properly
+	checkConnectionClose(t, connection)
+
+	// check if all expectations were met
+	checkAllExpectations(t, mock)
+}
```
Comment (juandspy) [inline storage_test.go:280]: No test with valid rows returned? `TestReadDisabledRules`

File: README.md [hunk 3]
```diff
@@ -64,17 +64,20 @@ contribute to this project.
 Usage of ./irae:
   -authors
         show authors
-  -output string
-        output to: CSV, S3
+  -check-s3-connection
+        check S3 connection and exit
+  -disabled-by-more-users
+         export rules disabled by more users
```
Comment (juandspy) [inline README.md:70]: I would say "export rules disabled by more than one user" instead

**Review by juandspy — CHANGES_REQUESTED** [review]
neatpicks

File: README.md (hunk 3 above)
Comment (tisnik) [reply README.md]: Fixed too

**Review by juandspy — APPROVED** [review]

**Review by Bee-lee — APPROVED** [review]
looks nice

**Review by joselsegura — APPROVED** [review]

**Review by dpensi — APPROVED** [review]

**Pull request merged**